study-builder/studies/brugada/snippets/picklist4.conf

```diff
@@ -0,0 +1,64 @@
+{
+  "picklistOptions": [
```

Comment:
can we have some meaningful file name that reflects what these options are rather than picklist4.conf

Reply:
renamed